scripts/atlas/README.md

```diff
@@ -3,79 +3,79 @@
+- `is_fragment` indicates whether the protein sequence is identified as a fragment in the MGnify90 sequence database.
+- `sequenceChecksum` is the CRC64 hash of the sequence. Can be used for cheaper lookups.
+- `esmfold_version` is the version of ESMFold, matching the model accessible as `esm.pretrained.esmfold_v{0,1}`
+- `atlas_version` is the Atlas version where this structure first appeared. Note: some of the predictions appearing for the first time in `v0` are also part of Atlas `v2023_02`.
 
-In parallel with `stats.parquet`, the sequences can be downloaded as fasta file from: <https://dl.fbaipublicfiles.com/esmatlas/v0/full/atlas.fasta>.
-The fasta file has `617051007` records matching the stats file, has file size 114GB, and has md5 hash `dc45f4383536c93f9d871facac7cca93`.
+Proteins longer than 1280 residues were not folded. To select only the records without missing structures use  `df[~df.plddt.isna()]`.
 
-We recommend using `s5cmd` or `aria2c` to download files (installable via anaconda).
-
-**To download any of the structures provided, please use this `aria2c` command**
+We recommend using `s5cmd` or `aria2c` to download files (can be installed via conda package manager).
+To download any of the structures provided, you can use this `aria2c` command:
 ```
 aria2c --dir=/path/to/download/to --input-file=url-file-provided.txt
 ```
 
-NOTE: Predicted Aligned errors will be uploaded at a later date, stay tuned!
 
-# High quality MGnify30 structures
+# Full database
+
+We separate the database by pTM and pLDDT.
+This will allow you to download structures based on your needs - for example you can choose to download only the most high confidence structures.
+Structures are provided by the bins given in this repo under [v0/full/bins.txt](v0/full/bins.txt) and [v2023_02/full/bins.txt](v2023_02/full/bins.txt).
+For example, the foldseek database containing ptm from 0.60 to 0.70 and plddt from 0.80 to 0.90 is named `tm_.60_.70_plddt_.80_.90.DB`.
+The data are given as bundles of 500k or 1M structures each.
+
+* **Bulk Predicted structures**: see `{v0,v2023_02}/full/tarballs/`. The URLs for all shards of a single bundle will be in e.g. [v0/full/tarballs/tm_.60_.70_plddt_.80_.90.txt](v0/full/tarballs/tm_.60_.70_plddt_.80_.90.txt). The URLs for all tarballs across bins and their shards is available under `{v0,v2023_02}/full/tarballs.txt`.
+* **Foldseek DBs** `{v0,v2023_02}/full/foldseekdb/` and `{v0,v2023_02}/full/foldseekdb.txt`
+* **Bulk Embeddings (NEW - coming soon)** under `{v0,v2023_02}/full/esm2_embeddings/` and `{v0,v2023_02}/full/esm2_embeddings.txt`.
+  * Note: Individual LM embeddings can be fetched via the API endpoint `/fetchEmbedding/ESM2/:id` as described on <https://esmatlas.com/about#api>
+
+Note as of March 16, 2022: We expect to release bulk embeddings, and limited backfill of v2023_02 structures and foldseekdb, by adding URLs to the respective files.
+At that time we will update the metadata.sqlite file as well.
```

Comment:
just realized the sqlite file was over 30G (54G), so I had to move it under `dl.fbaipublicfiles.com/large_objects/esmatlas/v2023_02/metadata-rc1.sqlite` instead. 

I also made a compressed sqlite file under the original directory, and uploaded to `dl.fbaipublicfiles.com/esmatlas/v2023_02/metadata-rc1.sqlite.gz`

Reply:
Great, updated the links now.